ui/Screen/Settings.svelte

```diff
@@ -1,15 +1,21 @@
 <script>
   import {
-    clear,
-    clearCache,
     settings,
     updateAppearance,
     updateRegistry,
   } from "../src/session.ts";
   import { networkOptions, themeOptions } from "../src/settings.ts";
 
-  import { Title, Text, Button } from "../DesignSystem/Primitive";
-  import { SidebarLayout, SegmentedControl } from "../DesignSystem/Component";
+  import Title from "../DesignSystem/Primitive/Title.svelte";
+
+  import Sidebar from "../Layout/Sidebar.svelte";
+
+  import Appearance from "./Settings/Appearance.svelte";
+  import Developer from "./Settings/Developer.svelte";
```

Comment:
Why wouldn't we have 1 component called `Settings/Item.svelte` where you have the header and a `<slot />` for the content.

Reply:
Seems to me like this isn't very DRY

Reply:
Yeah very good feedback and spot on, similar to https://github.com/radicle-dev/radicle-upstream/pull/582#discussion_r445397889.